apps/prairielearn/src/migrations/20230715221603_PLR_students.sql

```diff
@@ -4,7 +4,7 @@ CREATE TABLE IF NOT EXISTS PLR_students (
     --user_id BIGINT NOT NULL,
     display_name VARCHAR(256) NOT NULL,
     color VARCHAR(256),
-    course_instance_id BIGINT,
+    course_instance_id BIGINT NOT NULL,
```

Comment:
I was getting some issues with this here last time I tried `make deps` (because on PrairieLearn's admin section, users can be created without a `course_instance_id`). If we're going this route to easily populate while testing on their platform, should remove `NOT NULL`

Reply:
Yeah, Im not sure why that got added back. I was confused about why that was in the commit. It'll be removed.